### PR TITLE
💄 Support @Model display model pictures

### PR DESCRIPTION
**Pull request by Yanyutin753**
# Pull Request Checklist

### Note to first-time contributors: Please open a discussion post in [Discussions](https://github.com/open-webui/open-webui/discussions) and describe your changes before submitting a pull request.

**Before submitting, make sure you've checked the following:**

- [x] **Target branch:** Please verify that the pull request targets the `dev` branch.
- [x] **Description:** Provide a concise description of the changes made in this pull request.
- [x] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [x] **Testing:** Have you written and run sufficient tests for validating the changes?
- [x ] **Code review:** Have you performed a self-review of your code, addressing any coding standard issues and ensuring 
# Changelog Entry

### Description

💄 Support @Model display model pictures

### Added

💄 Support @Model display model pictures

### Changed

💄 Support @Model display model pictures

### Screenshots or Videos

image

e6435264db20d2e1e70e8acb1d24139

### CODE REVIEW

File: src/lib/components/chat/MessageInput/Models.svelte [hunk 1]
```diff
@@ -21,8 +21,10 @@
 	let filteredModels = [];
 
 	$: filteredModels = $models
-		.filter((p) =>
-			p.name.toLowerCase().includes(prompt.toLowerCase().split(' ')?.at(0)?.substring(1) ?? '')
+		.filter(
```
Comment (tjbck) [inline src/lib/components/chat/MessageInput/Models.svelte:24]: Should not change the default behaviour

File: src/lib/components/chat/MessageInput/Models.svelte [hunk 2]
```diff
@@ -159,7 +161,12 @@
 								}}
 								on:focus={() => {}}
 							>
-								<div class=" font-medium text-black dark:text-gray-100 line-clamp-1">
+								<div class="flex font-medium text-black dark:text-gray-100 line-clamp-1">
+									<img
+										src={model?.info?.meta?.profile_image_url ?? '/static/favicon.png'}
+										alt="Model"
```
Comment (tjbck) [inline src/lib/components/chat/MessageInput/Models.svelte:167]: alt tag should be set to model name

File: src/lib/components/chat/MessageInput/Models.svelte (hunk 1 above)
Comment (Yanyutin753) [reply src/lib/components/chat/MessageInput/Models.svelte]: okok，Thanks!

File: src/lib/components/chat/MessageInput/Models.svelte (hunk 2 above)
Comment (Yanyutin753) [reply src/lib/components/chat/MessageInput/Models.svelte]: okok，Thanks!

Comment (Yanyutin753) [reply src/lib/components/chat/MessageInput/Models.svelte]: I will try it later

**Comment by Yanyutin753** [comment]
OK @tjbck

**Comment by tjbck** [comment]
Thanks!

**Pull request merged**